test/test_generator.py

```diff
@@ -416,12 +416,7 @@ def test_rag_token_generator(rag_generator):
 
 @pytest.mark.slow
```

Comment:
Are we still using the `slow` and `generator` markers?

Reply:
It can be still helpful to run only a "subset" of tests locally. As I don't see any harm in keeping them, I would leave them for now.